src/services/inplayer.account.service.ts

```diff
@@ -52,6 +52,34 @@ export const getUser = async (): Promise<Customer> => {
 
 export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
+export const updateCustomer: UpdateCustomer = async (values) => {
+  try {
+    const consents: { [key: string]: string } = {};
+    values.consents?.map((consent: CustomerConsent) => {
```

Comment:
It doesn't look like the Consents are set via this path anywhere.  I think it would be better to do a different `InPlayer.Account.updateAccount` call in a `updateCustomerConsents` method for the inplayer service.  They are saved separately anyhow.

The other approach would be to keep updateCustomer like this, but then refactor the Cleeng service to make 2 calls (1 to cleeng's update customer endpoint and 1 to cleeng's update consents endpoint.)  However, since the save calls are independent anyhow I lean towards keeping the 2 methods (`updateCustomer` and `updateCustomerConsents`.)  This also is less refactoring and gives a path towards adding customer consents as a top level resource if that becomes necessary instead of saving the data in customer fields.

Reply:
because `fullName` is a required field for InPlayer `updateCustomer`, we have to use this method to update the user consents or any other user extras